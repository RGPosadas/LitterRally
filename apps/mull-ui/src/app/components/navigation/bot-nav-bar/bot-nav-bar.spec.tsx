import React from 'react';
import { render } from '@testing-library/react';

import BottomNavBar from './bot-nav-bar';
import { Router } from 'react-router-dom';

import { createMemoryHistory } from 'history';

import renderer from 'react-test-renderer';

describe('Header', () => {
  it('should render successfully', () => {
    const history = createMemoryHistory();

    const { baseElement } = render(
      <Router history={history}>
        <BottomNavBar />
      </Router>
    );

    expect(baseElement).toBeTruthy();
  });

  it('should match snapshot', () => {
    const history = createMemoryHistory();

    const tree = renderer
      .create(
        <Router history={history}>
          <BottomNavBar />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
