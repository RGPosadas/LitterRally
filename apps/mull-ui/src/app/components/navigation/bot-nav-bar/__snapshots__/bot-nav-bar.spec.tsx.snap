// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BotNavBar should match snapshot 1`] = `
<div
  className="bot-nav-container"
>
  <div
    className="nav-buttons-container"
  >
    <a
      aria-current={null}
      data-testid="home-navlink"
      href="/home"
      onClick={[Function]}
    >
      <svg
        className="nav-button"
      >
        HomeIcon.svg
      </svg>
    </a>
    <a
      aria-current={null}
      data-testid="map-navlink"
      href="/map"
      onClick={[Function]}
    >
      <svg
        className="nav-button"
      >
        MapIcon.svg
      </svg>
    </a>
    <a
      aria-current={null}
      data-testid="create-event-navlink"
      href="/create-event"
      onClick={[Function]}
    >
      <svg
        className="nav-middle-button"
      >
        CreateEventIcon.svg
      </svg>
    </a>
    <a
      aria-current={null}
      data-testid="tools-navlink"
      href="/tools"
      onClick={[Function]}
    >
      <svg
        className="nav-button"
      >
        MachineLearningIcon.svg
      </svg>
    </a>
    <a
      aria-current={null}
      data-testid="messages-navlink"
      href="/messages"
      onClick={[Function]}
    >
      <svg
        className="nav-button"
      >
        MessagesIcon.svg
      </svg>
    </a>
  </div>
</div>
`;
